### DISCUSSION TITLE
Interesting looking ppg data (collected with emotibit)

### DISCUSSION THREAD

**Discussion by ejpretty** [neuropsychology/NeuroKit#914] [Q&A]
Hi all! 

I am quite new to this whole thing but I have just made a bit of a processing pipeline for some PPG and whilst some of it looks okay(?), I'm getting some funny looking plots and some extreme values (on multiple participants for this experiment) that aren't being corrected by any cleaning method (in fact, the nabian2018 method isn't working for me at all).

I used an emotibit to collect this data, and so I'd love some advice on where to go from here!

Thanks in advance :)
![Capture_ppg](https://github.com/neuropsychology/NeuroKit/assets/66713921/d89c5823-1d31-4c13-8d0f-4a73aa2b727a)

**Comment by DominiqueMakowski** [comment]
The main limitation is the low sampling rate of your data, which limits what you can do with it. 

Maybe first you can rescale / standardize your raw signal (nk.standardize). This shouldn't change anything, but in the resulting plot you will have the raw signal with roughly the same values as the cleaned one, and you'll be able to better see the effects of various cleanings. 

At least your average heart rate is plausible, and the overlays shows that  "most" of the "peaks" seemed to have been captured. So it's not completely off, but for applications like heart rate variability, well there's only so much you can do with a low sampling rate
